src/app/components/sandbox/CodeEditor/CodeMirror.js

```diff
@@ -410,8 +410,9 @@ export default class CodeEditor extends React.PureComponent<Props, State> {
     }
 
     if (preferences.lintEnabled) {
+      const initialized = 'eslint' in window;
       System.import('app/utils/codemirror/eslint-lint')
-        .then(initializer => initializer.default())
+        .then(initializer => !initialized && initializer.default())
```

Comment:
`eslint-lint.js`.`initialize()` (the default export) should execute only when importing eslint the first time. This gets triggered whenever switching between CodeMirror and Monaco.